portal/views/assessment_engine.py

```diff
@@ -1719,7 +1719,7 @@ def patient_assessment_status(patient_id):
       - application/json
     responses:
       200:
-        description: return current overall assessment status of given patient
+        description: return current assessment status of given patient
```

Comment:
Not necessary, but it would be nice to detail what we're returning - obviously this comment only covers the generic idea.

Reply:
thanks, I'll have to add an enum for the different "status" values, and a references for the Q/NR values